opencensus/stats/testing/test_utils.cc

```diff
@@ -33,17 +33,32 @@ namespace testing {
 ViewData TestUtils::MakeViewData(
     const ViewDescriptor& descriptor,
     std::initializer_list<std::pair<std::vector<std::string>, double>> values) {
+  std::vector<std::tuple<std::vector<std::string>, double, absl::Time>> params;
+  for (const auto& value : values) {
+    params.push_back(
+        std::make_tuple(value.first, value.second, absl::UnixEpoch()));
+  }
+  return TestUtils::MakeViewData(descriptor, params);
+}
+
+// static
+ViewData TestUtils::MakeViewData(
+    const ViewDescriptor& descriptor,
+    std::vector<std::tuple<std::vector<std::string>, double, absl::Time>>
+        values) {
   auto impl = absl::make_unique<ViewDataImpl>(absl::UnixEpoch(), descriptor);
   std::vector<BucketBoundaries> boundaries = {
       descriptor.aggregation().bucket_boundaries()};
   for (const auto& value : values) {
     MeasureData measure_data = MeasureData(boundaries);
-    measure_data.Add(value.second);
-    impl->Merge(value.first, measure_data, absl::UnixEpoch());
+    measure_data.Add(std::get<1>(value));
+    impl->Merge(std::get<0>(value), measure_data, std::get<2>(value));
   }
   if (impl->type() == ViewDataImpl::Type::kStatsObject) {
+    std::cerr << "here0\n";
```

Comment:
Leftover debugging. Also below.

Reply:
Should be all removed now. I had to do some more debugging for a flakey test. Please double check again that I didn't leave anything there. :)